spring-petclinic-microservices/spring-petclinic-customers-service/src/main/resources/application.properties

```diff
@@ -4,17 +4,16 @@ spring.cloud.azure.profile.tenant-id=${AZURE_KEYVAULT_TENANTID}
 spring.cloud.azure.credential.client-id=${AZURE_KEYVAULT_CLIENTID}
 spring.cloud.azure.credential.client-secret=${AZURE_KEYVAULT_CLIENTKEY}
 spring.cloud.azure.keyvault.secret.property-source[0].refresh-interval=1800000
-spring.cloud.azure.keyvault.secret.property-source[0].secret-keys=cosmosdburi,cosmosdbkey,cosmosdbsecondarykey,redisuri,redispassword
+spring.cloud.azure.keyvault.secret.property-source[0].secret-keys=cosmosdburi,cosmosdbkey,redisuri,redispassword
```

Comment:
also need update setup-env-variables-azure-template.sh here
since you delete cosmosdbsecondarykey

Reply:
I think we can directly change application.properties to application.yaml now, since you have delete CustomersAppConfiguration.java